lightsail-instance-launch-scripts/windows_server/enable_win_narrator.bat

```diff
@@ -0,0 +1,9 @@
+<script>
+    net start audiosrv
+
+    reg add "HKCU\Software\Microsoft\Windows NT\CurrentVersion\Accessibility" /t REG_SZ /v Configuration /d narrator /f
+
```

Comment:
all these line breaks are needed?

Reply:
I can remove or add inline short comments

Reply:
example 

<script>
    #start audio service
    net start audiosrv
    #set reg key to start narrator at logon
    reg add "HKCU\Software\Microsoft\Windows NT\CurrentVersion\Accessibility" /t REG_SZ /v Configuration /d narrator /f
    #start narrator
    narrator
</script>

Reply:
ah I thought that last line was part of the registry add thing. thanks for explaining.

Reply:
added the inline comments. that should help readability.